modules/awae/src/AWAE.f90

```diff
@@ -54,6 +56,10 @@ module AWAE
    public :: AWAE_TEST_CalcOutput
    public :: AWAE_TEST_Interp2D
 
+   real(ReKi), parameter, private :: I33(3,3) = &
```

Comment:
this is a little nicer than calling the `Eye` function to generate the identity.

Reply:
Let's remove that.  We can also address the comment on line 1203 about the hub and disk avg vel - we don't call `IfW_CalcOutput` anymore.  Although, I'm a little unsure if that will cause issues in IfW initialization if that is random junk for the hub orientation (I don't think we check the dcm, so it might be fine).